Dockerfile

```diff
@@ -10,16 +10,11 @@ COPY requirements.txt .
 COPY requirements_test.txt .
 COPY ocrd_tesserocr ./ocrd_tesserocr
 COPY Makefile .
-RUN apt-get update && \
-    apt-get install -y software-properties-common && \
-    add-apt-repository ppa:alex-p/tesseract-ocr && \
-    apt-get update && \
-    apt-get -y install --no-install-recommends \
-    libtesseract-dev \
-    tesseract-ocr \
+RUN make deps-ubuntu && \
+    apt-get install -y --no-install-recommends \
+    g++ \
     tesseract-ocr-script-frak \
     tesseract-ocr-deu \
-    build-essential \
     && make deps install \
     && rm -rf /build-ocrd \
-    && apt-get -y remove --auto-remove build-essential
+    && apt-get -y remove --auto-remove g++ libtesseract-dev make
```

Comment:
Wait, why would you remove `libtesseract-dev`? That will break `tesserocr` and `ocrd-tesserocr`, won't it?

Reply:
Yes, removing the headers and library will break tesserocr at runtime i think.

Reply:
It's already handled by `make deps-ubuntu`.

Reply:
I have tested it and it is not a problem at runtime, works fine.

Reply:
Ok great – thanks for testing!

(It looks like all runtime dependencies are in `libtesseract4` already.)